pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -49,8 +49,8 @@ func TestFeatureToggleFiles(t *testing.T) {
 			if flag.Name != strings.TrimSpace(flag.Name) {
 				t.Errorf("flag Name should not start/end with spaces.  See: %s", flag.Name)
 			}
-			if flag.AllowSelfServe && flag.Stage != FeatureStageGeneralAvailability {
-				t.Errorf("only allow self-serving GA toggles")
+			if flag.AllowSelfServe && !(flag.Stage == FeatureStageGeneralAvailability || flag.Stage == FeatureStagePublicPreview || flag.Stage == FeatureStageDeprecated) {
```

Comment:
@jcalisto Could you please take a look at this? I aligned this to the changes of https://github.com/grafana/grafana/pull/81562.

Reply:
LGTM, thanks for updating this check @mgyongyosi!